django_q/tests/test_monitor.py

```diff
@@ -10,15 +11,16 @@
 
 @pytest.mark.django_db
 def test_monitor(monkeypatch):
-    assert Stat.get(0).sentinel == 0
+    cluster_id = uuid.uuid4()
+    assert Stat.get(pid=0, cluster_id=4).sentinel == 0
```

Comment:
`cluster_id=4` is this a typo?

Reply:
Yeah, totally, it's a typo.  Should be `cluster_id=cluster_id`.